Batteries/Data/List/Basic.lean

```diff
@@ -1393,3 +1393,6 @@ where
   | [], r, t => reverseAux t r
   | l, [], t => reverseAux t l
   | a::l, b::r, t => bif s a b then loop l (b::r) (a::t) else loop (a::l) r (b::t)
+
+/-- `finRange n` is the list of all elements of `Fin n` in order -/
```

Comment:
In mathlib the docstring is
```
/-- All elements of `Fin n`, from `0` to `n-1`. The corresponding finset is `Finset.univ`. -/
```
Should the last sentence be kept, with an added "in mathlib"?

Reply:
Good question. I'm not sure. How useful is the second sentence for Mathlib users? It seems like it is a trivial conclusion from the first sentence, isn't it?